src_app/test_scenes/scenePlaneDetector.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { PlaneDetector } from "magic-script-components";
+
+class ScenePlaneDetector extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { points: {} };
+  }
+
+  componentDidMount() {
+    this.planeDetector = PlaneDetector.instance();
+    this.planeDetector.addOnPlaneDetectedObserver(this, this.onPlaneDetected);
+    this.planeDetector.addOnPlaneUpdatedObserver(this, this.onPlaneUpdated);
+    this.planeDetector.addOnPlaneRemovedObserver(this, this.onPlaneRemoved);
+    this.planeDetector.addOnPlaneTappedObserver(this, this.onPlaneTapped);
+    this.planeDetector.startDetecting({
+      planeType: ["horizontal", "vertical"]
+    });
+
+    this.planeDetector.getAllPlanes({}, (error, result) => {
+      if (error) {
+        console.log("getAllPlanes - error", result);
+      } else {
+        console.log("getAllPlanes", result);
+      }
+    });
+  }
+
+  onPlaneDetected = event => {
+    console.log("onPlaneDetected", event);
+    // const { points } = this.state;
+    // points[event.id] = event.vertices;
```

Comment:
Why does the event handlers are commented out? That means that this test scene does not work, does it?